buddyboss/buddyboss-platform
# Added new icons for TikTok, Telegram and ClubHouse

### All Submissions:


Fixes #2450 

### How to test the changes in this Pull Request:

1. Add links to newly added fields (TikTok, Telegram and ClubHouse) theme options -> Footer -> Social Links
2. Check icons in footer

### Connected PR in Theme
https://github.com/buddyboss/buddyboss-theme/pull/1465

### Proof Screenshots or Video
![image](https://user-images.githubusercontent.com/53166220/120589837-219bf580-c457-11eb-97d9-3a6a13305bf9.png)


### Other information:
